index.html

```diff
@@ -0,0 +1,172 @@
+                <p>Platform release: crunchyroll - wit studio NHK</p>
+            </div>
+        </section>
+        <section id="section1" class="section1">
+            <div class="cont-ti">
+                <h2>Main Program</h2>
+            </div>
+            <hr class="line">
+            <ul class="card-container">
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr1.png" alt="">
+                        <h3>Exhibition</h3>
+                        <p>Discover the "Latest Releases" in movies and series, find the most popular productions and
+                            receive recommendations.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr2.png" alt="">
+                        <h3>Forum</h3>
+                        <p>Participa en "Discusiones generales", analiza episodios, especula sobre tramas, lee reseñas
+                            de anime y comparte fanart y cosplay.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr3.png" alt="">
+                        <h3>Network</h3>
+                        <p> Create your profile, send private messages, join groups, check the events calendar and find
+                            friends with similar interests.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr4.png" alt="">
+                        <h3>Lecture</h3>
+                        <p>Browse the schedule, meet guest speakers, participate in Q&A sessions, and enjoy live
+                            streams.</p>
+                    </div>
+                </li>
+                <li>
+                    <div class="card">
+                        <img src="ressources/icons/pr5.png" alt="">
+                        <h3>Workshop</h3>
+                        <p>Learn with featured instructors, sign up for specific times, and access additional resources
+                            and materials.</p>
+                    </div>
+                </li>
+            </ul>
+            <div class="cont-des">
+                <h2 class="mo"><a href="#">SEE MORE</a></h2>
+                <button type="button" class="des">Join into the world of anime</button>
+            </div>
+        </section>
+        <section class="section2">
+            <div class="title-section2">
+                <h2>Red carpet</h2>
+            </div>
+            <hr class="line">
+            <section>
+                <ul id="new-animes"></ul>
+            </section>
+            <div class="button-section2">
+                <button type="button">MORE<img src="ressources/icons/down.png"></button>
+            </div>
```

Comment:
- [ ] please like clearly stated on the guidelines and like the previous reviewer mentioned, we are to follow the Behance design be it positioning and design of elements. We are allowed to personalize the content. Kindly  the speaker section is in three rows and two columns just like the Behance design

yours
![Screenshot (1228)](https://github.com/AndresEspi/animePage/assets/65298048/afc2aa7c-3748-4d26-8310-7b6998a2f646)
original
![Screenshot (1078)](https://github.com/AndresEspi/animePage/assets/65298048/01015411-5cf1-4c7f-a115-f19ddf088742)

Reply:
Please, can you review this point again since I corrected it before? Please, I await your response to be able to load the project again.